azure-pipelines.yml

```diff
@@ -2,8 +2,38 @@
 # Build, test, and archive an Xcode workspace on macOS.
 # Add steps that install certificates, test, sign, and distribute an app, save build artifacts, and more:
 # https://docs.microsoft.com/azure/devops/pipelines/languages/xcode
+
 jobs:
-  - template: ./ci-jobs/functional_test.yml
+  - job: 'unitTests'
+
+    pool:
+      vmImage: 'ubuntu-18.04'
+    strategy:
+      matrix:
+        Python37:
+          python.version: '3.7'
+        Python38:
+          python.version: '3.8'
+        Python39:
+          python.version: '3.9'
+
+    steps:
+    - task: UsePythonVersion@0
+      displayName: 'Use Python $(python.version)'
+      inputs:
+        versionSpec: '$(python.version)'
+
+    - script: pip install tox
+      displayName: 'Install Tox'
+
+    - script: tox -e py
```

Comment:
isn't tox already running the tests for each python version from its config or I just don't understand something?